pkgs/applications/virtualization/nixpacks/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nixpacks";
+  version = "v0.1.7";
+  doCheck = true;
+  cargoSha256 = "iA8ODh1NmUBfyH6syvVD32jDeLYG2LeLTo2lQoO93lc=";
+  nativeBuildInputs = [ clippy ];
```

Comment:
This and `doCheck` must go, other than these LGTM.

Reply:
oh, I really missed these lines!